data_steward/cdr_cleaner/cleaning_rules/unit_normalization.py

```diff
@@ -175,14 +178,14 @@ def setup_rule(self, client=None):
         # Uploading data to _unit_mapping table
         unit_mappings_csv_path = os.path.join(resources.resource_files_path,
                                               UNIT_MAPPING_FILE)
-        job = bq.upload_csv_data_to_bq_table(client, self.dataset_id,
+        job = bq.upload_csv_data_to_bq_table(client, self.sandbox_dataset_id,
```

Comment:
This is not within the scope of the ticket, however, [`bq.upload_csv_data_to_bq_table` ](https://github.com/all-of-us/curation/blob/6049de9db6ad8ff5db1ad80a086421e3d9c8944e/data_steward/utils/bq.py#L117)returns the job results object instead of the job object

Reply:
@ChaoPang I added a doc string about the errors that could be raised via the `bq.upload_csv_data_to_bq_table` helper function.  I changed the variable name `job` to `result`.  I removed the erroneous call to `job.result()`.

Reply:
@lrwb-aou thanks for adding the docstring!